net.lutris.Lutris.yml

```diff
@@ -0,0 +1,364 @@
+id: net.lutris.Lutris
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+command: lutris
+rename-icon: lutris
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+  - --allow=multiarch
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gnome.Mutter.DisplayConfig
+  - --talk-name=ca.desrt.dconf
+  - --filesystem=xdg-run/dconf
+  - --filesystem=xdg-config/dconf:ro
+  - --filesystem=~/Games:create
+  - --persist=~/.wine
+  - --filesystem=xdg-data/Steam:ro
+  - --filesystem=~/.var/app/com.valvesoftware.Steam:ro
```

Comment:
Has this been tested? I've noted Flatpak doesn't allow apps to see each other's sandboxes even with --filesystem=home.

Reply:
Yes, it's working as intended. Gnome Games does [similar thing](https://github.com/flathub/org.gnome.Games/blob/1be076089e333bccc2d767be6d352e7551a767f3/org.gnome.Games.json#L37).